build-system/scripts/can_use_ci_cache

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
 set -euo pipefail
-type docker &> /dev/null && docker ps 2>&1 > /dev/null && [ -f ~/.aws/credentials ] && ecr_login > /dev/null || exit 1
\ No newline at end of file
+if [ -n $AWS_ACCESS_KEY_ID ] && [ -n $AWS_SECRET_ACCESS_KEY ]; then
```

Comment:
Nice

Reply:
it'd be nice to also get this into earthly-local

Reply:
also not sure everyone is comfortable having their creds in env vars

Reply:
Tbf we shouldn't need creds to be in the env since we're usign the `aws` cli. It picks them up automatically from its credentials file. I'll remove this

Reply:
yeah just unfortunately not the case for earthly-local

Reply:
but that's out of scope, just nice to have in the future

Reply:
yep, I did extract the creds in the last commit inside earthly-local 😬

Reply:
Perfect! Not sure why the grimace :)